app/utils/colors.ts

```diff
@@ -0,0 +1,4 @@
+export const colors = {
+  field: '#f3f3f3',
+  line: '#aadd33',
+};
```

Comment:
please use uppercase for constants like this
COLORS {
name of color you can check here https://chir.ag/projects/name-that-color/#6195ED